prow/plugins.yaml

```diff
@@ -222,6 +222,29 @@ repo_milestone:
     maintainers_id: 2460384
     maintainers_team: kubernetes-milestone-maintainers
 
+project_config:
+  project_org_configs:
+    kubernetes:
+      org_maintainers_team_id: 2460384
```

Comment:
nit: could be

```yaml
org_maintainers_team:
  id:
  name:
```

With this, it looks like you could override the team ID but not the team name with a repo config? Is that something we want?

Reply:
Yes that's true. When checking a user against the team only the team ID is used. I will remove the team name from the config